confgenerator/testdata/valid/linux/metrics-receiver_oracledb_unix_socket/golden_otel.yaml

```diff
@@ -0,0 +1,1113 @@
+        label: GLOBAL_NAME
+        new_label: global_name
+      - action: update_label
+        label: INSTANCE_ID
+        new_label: instance_id
+      - action: update_label
+        label: TABLESPACE_NAME
+        new_label: tablespace_name
+      - action: update_label
+        label: CONTENTS
+        new_label: contents
+      - action: update_label
+        label: STATUS
+        new_label: status
+      - action: update_label
+        label: PROGRAM
+        new_label: program
+      - action: update_label
+        label: WAIT_CLASS
+        new_label: wait_class
+  metricstransform/otel_1:
+    transforms:
+    - action: update
+      include: otelcol_process_uptime
+      new_name: agent/uptime
+      operations:
+      - action: toggle_scalar_data_type
+      - action: add_label
+        new_label: version
+        new_value: google-cloud-ops-agent-metrics/latest-build_distro
+      - action: aggregate_labels
+        aggregation_type: sum
+        label_set:
+        - version
+    - action: update
+      include: otelcol_process_memory_rss
+      new_name: agent/memory_usage
+      operations:
+      - action: aggregate_labels
+        aggregation_type: sum
+        label_set: []
+    - action: update
+      include: otelcol_grpc_io_client_completed_rpcs
+      new_name: agent/api_request_count
+      operations:
+      - action: toggle_scalar_data_type
+      - action: update_label
+        label: grpc_client_status
+        new_label: state
+      - action: aggregate_labels
+        aggregation_type: sum
+        label_set:
+        - state
+    - action: update
+      include: otelcol_googlecloudmonitoring_point_count
+      new_name: agent/monitoring/point_count
+      operations:
+      - action: toggle_scalar_data_type
+      - action: aggregate_labels
+        aggregation_type: sum
+        label_set:
+        - status
+    - action: update
+      include: ^(.*)$$
+      match_type: regexp
+      new_name: agent.googleapis.com/$${1}
+  normalizesums/oracledb_oracledb_0: {}
+  resourcedetection/_global_0:
+    detectors:
+    - gce
+receivers:
+  hostmetrics/default__pipeline_hostmetrics:
+    collection_interval: 60s
+    scrapers:
+      cpu: {}
+      disk: {}
+      filesystem: {}
+      load: {}
+      memory: {}
+      network: {}
+      paging: {}
+      process:
+        mute_process_name_error: true
+      processes: {}
+  prometheus/fluentbit:
+    config:
+      scrape_configs:
+      - job_name: logging-collector
+        metrics_path: /metrics
+        scrape_interval: 1m
+        static_configs:
+        - targets:
+          - 0.0.0.0:20202
+  prometheus/otel:
+    config:
+      scrape_configs:
+      - job_name: otel-collector
+        scrape_interval: 1m
+        static_configs:
+        - targets:
+          - 0.0.0.0:20201
+  sqlquery/oracledb_oracledb:
+    collection_interval: 60s
+    datasource: oracle://:@localhost:1521/?SID%3Ddb19c%26UNIX%2BSOCKET%3D%252Ftmp%252Foracle%252Fsocket
```

Comment:
Why does the query portion of the URI need to be URI-escaped? The only thing I can find documented on this syntax is [here](https://github.com/sijms/go-ora#servers-url-options) which suggests that the `&` and `=` key/value separators should not be URI-escaped.

Reply:
Ah, looks like I double escaped the full query string, I'll fix that.